terminal_client.go

```diff
@@ -119,7 +119,7 @@ func (c *TerminalClient) listCmd() error {
 
 	tbl := TableDef{
 		Columns: []ColumnDef{
-			ColumnDef{"id", 25},
+			ColumnDef{"id", 42},
```

Comment:
The change in column length was not due to the active marker but rather the fact that the new system of <adjective>-<city name> was making the deploy ID wrap on to the next line which was annoying for copying IDs. For example, 
This:
```
Deploys:
id                        pid   tracked port st  messages                                           
charming-honolulu-2016-04     0 y       8001   0 []                                                 
-22-05-13-01                                                                                        
creepy-las-vegas-2016-04-     0 y       8002   0 []                                                 
22-00-49-28                                                                                         
exciting-lagos-2016-04-22     0 y       8005   0 []                                                 
-00-51-44                                                                                           
expensive-richmond-2016-0     0 y       8004   0 []                                                 
4-22-05-00-07                                                                                       
serene-lahore-2016-04-29-     0 n          0   0 []                                                 
05-04-05                                                                                            
unique-lahore-2016-04-22-     0 y       8006   0 []                                                 
05-28-58                                                                                            
unique-lahore-2016-04-29-     0 n          0   0 []                                                 
05-00-03                                                                                            
vibrant-calgary-2016-05-0     0 n          0   0 []                                                 
4-00-31-11     
```

Becomes:
```
Deploys:
id                                         pid   tracked port st  messages                                           
charming-honolulu-2016-04-22-05-13-01          0 y       8001   0 []                                                 
creepy-las-vegas-2016-04-22-00-49-28           0 y       8002   0 []                                                 
exciting-lagos-2016-04-22-00-51-44             0 y       8005   0 []                                                 
expensive-richmond-2016-04-22-05-00-07         0 y       8004   0 []                                                 
serene-lahore-2016-04-29-05-04-05              0 n          0   0 []                                                 
unique-lahore-2016-04-22-05-28-58              0 y       8006   0 []                                                 
unique-lahore-2016-04-29-05-00-03              0 n          0   0 []                                                 
vibrant-calgary-2016-05-04-00-31-11            0 n          0   0 []    
```

Currently the active marker appears after the messages like so:
```
Deploys:
id                                         pid   tracked port st  messages                                           
charming-honolulu-2016-04-22-05-13-01          0 y       8001   0 []                                                 
creepy-las-vegas-2016-04-22-00-49-28           0 y       8002   0 []                                                 
exciting-lagos-2016-04-22-00-51-44             0 y       8005   0 []                                                 
expensive-richmond-2016-04-22-05-00-07         0 y       8004   0 []                                                 
serene-lahore-2016-04-29-05-04-05              0 n          0   0 []                                                 
unique-lahore-2016-04-22-05-28-58              0 y       8006   0 []                                                 
unique-lahore-2016-04-29-05-00-03              0 n          0   0 []                                                 
vibrant-calgary-2016-05-04-00-31-11        30554 y       8007 200 [] <--- ACTIVE                                     
haproxy-8098                               30572 n       8098   0 []                                                 
haproxy-8099                               30572 n       8099   0 [] 
```
Would it be better to change to using an asterisk?